packages/dapp/src/stores/settings/useSettings.ts

```diff
@@ -1,16 +0,0 @@
-import { SettingsState } from '@transferto/shared/src/types';
-import { shallow } from 'zustand/shallow';
-import { useSettingsStore } from './SettingsStore';
-
-export const useSettings = <K extends keyof SettingsState>(
-  keys: Array<K>,
-): Pick<SettingsState, (typeof keys)[number]> => {
-  return useSettingsStore(
-    (state) =>
-      keys.reduce((values, key) => {
-        values[key] = state[key];
-        return values;
-      }, {} as Pick<SettingsState, (typeof keys)[number]>),
-    shallow,
-  );
-};
```

Comment:
@chybisov Do we need any of this as of now? I found this in the widget repo but did not have any use for it. Let me know, if we should rather get this back in place.